lib/plugins/aws/package/compile/events/cloudFront/index.js

```diff
@@ -0,0 +1,305 @@
+'use strict';
+
+const _ = require('lodash');
+const url = require('url');
+const chalk = require('chalk');
+const ServerlessError = require('../../../../../../classes/Error');
+
+class AwsCompileCloudFrontEvents {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options;
+    this.provider = this.serverless.getProvider('aws');
+    this.hooks = {
+      'before:package:compileFunctions': this.prepareFunctions.bind(this),
+      'package:compileEvents': this.compileCloudFrontEvents.bind(this),
+    };
+    if (this.serverless.processedInput.commands[0] === 'remove') {
+      const message = [
+        "Don't forget to manually remove your Lambda@Edge functions ",
+        'once the CloudFront distribution removal is successfully propagated!',
+      ].join('');
+      this.serverless.cli.log(message, 'Serverless', { color: 'orange' });
```

Comment:
If I see correctly, it's output also if there are no ClouFront endpoints involved (?)

To avoid unnecessary noise, it'll be cleaner to output that only if there were any CF endpoints in a stack.

Really solid would be if we also warn if single CF  endoint is removed (so also in special update cases). Still I imagine it'll be more time-taking, and I guess it'll be worth to come up with some generic handler (so other plugins can benefit from such inspection as well). So if at all, I would leave it for other PR

Reply:
Yes, that was the initial intention. Good catch. Will update in a minute...